sys/sema/sema.c

```diff
@@ -11,169 +12,109 @@
  *
  * @file
  *
- * @author  Christian Mehlis <mehlis@inf.fu-berlin.de>
- * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
- * @author  René Kijewski <kijewski@inf.fu-berlin.de>
+ * @author  Víctor Ariño <victor.arino@zii.aero>
```

Comment:
Keeping us in the Copyright but removing our authorship? ;-)

(though given the amount of code left I understand the latter step)